src/screens/AppIntroScreen.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect } from 'react';
+import { ListRenderItem, ScrollView, StyleSheet, View } from 'react-native';
+import AppIntroSlider from 'react-native-app-intro-slider';
+
+import { BoldText, Image, RegularText, SafeAreaViewFlex, Wrapper } from '../components';
+import { colors, normalize } from '../config';
+import { useStaticContent } from '../hooks';
+import { parseIntroSlides } from '../jsonValidation';
+import { AppIntroSlide } from '../types';
+
+const keyExtractor = (slide: AppIntroSlide) => slide.title + slide.text;
+
+const SliderButton = ({ label }: { label: string }) => {
+  return (
+    <View style={styles.underline}>
+      <BoldText>{label.toUpperCase()}</BoldText>
+    </View>
+  );
+};
+
+type Props = {
+  setOnboardingComplete: () => void;
+};
+
+const renderSlide: ListRenderItem<AppIntroSlide> = ({ item }) => {
+  return (
+    <ScrollView>
+      <Image
```

Comment:
what about

<img width="1167" alt="Bildschirmfoto 2021-11-24 um 12 48 46" src="https://user-images.githubusercontent.com/1942953/143233145-a2cfc379-5da1-4f45-b7f2-11181e0a224e.png">

because of mixing js and ts?

Reply:
yes.
it occured when we stopped destructuring `source` from `props` in `image`.

we did not notice it back then, and i did not find the right place to change it yet.

Reply:
ideas, what you would do and how long it could take?

Reply:
it is probably a matter of finding a decent name for the destructured `source.props`. we tried already and failed, that is why we ended up with the solution of not destructuring it. which then causes type inference to malfunction.

changing it to be a typescript file would also be a solution, but i would need to check how many issues that brings with it, due to other things not being typed yet

Reply:
ok #410